python/cugraph/cugraph/sampling/random_walks.py

```diff
@@ -32,19 +47,25 @@ def uniform_random_walks(G, start_vertices, max_depth):
 
 
 def random_walks(
-    G,
-    random_walks_type="uniform",
-    start_vertices=None,
-    max_depth=None,
-    use_padding=False,
-    legacy_result_type=True,
-):
+    G: Graph,
+    random_walks_type: str = "uniform",
+    start_vertices: Union[int, list, cudf.Series, cudf.DataFrame] = None,
+    max_depth: int = None,
+    use_padding: bool = False,
+    legacy_result_type: bool = True,
+) -> Tuple[cudf.Series, cudf.Series, Union[None, int, cudf.Series]]:
     """
-    # FIXME: make the padded value for vertices with outgoing edges
-    # consistent in both SG and MG implementation.
-    compute random walks for each nodes in 'start_vertices' and returns
+    Compute random walks for each nodes in 'start_vertices' and returns
     either a padded or a coalesced result. For the padded case, vertices
-    with no outgoing edges will be padded with NA
+    with no outgoing edges will be padded with -1.
+
+    When 'use_padding' is turned OFF, 'random_walks' returns a coalesced
+    result which is a compressed version of the padded one. In the padded
+    form, sources with no out_going edges are padded with -1s in the
+    'vertex_paths' array and their corresponding edges('edge_weight_paths')
+    with 0.0s (given 'legacy_result_type' is turned ON). If 'legacy_result_type'
```

Comment:
Instead of saying "(given 'legacy_result_type' is turned ON)" at the end of this sentence, maybe the start of the sentence should be:
"When 'legacy_result_type' is True and 'use_padding' is False..."